tests/testthat/test-sce_to_anndata.R

```diff
@@ -20,7 +23,23 @@ test_that("Conversion of SCE to AnnData works as expected", {
   expect_equal(colnames(colData(sce)), colnames(colData(converted_sce)))
   expect_equal(colnames(rowData(sce)), colnames(rowData(converted_sce)))
 
+  # test that H5 file is created with new assay name
+  # add logcounts
+  logcounts(sce) <- counts(sce)
+  new_anndata_file <- "test_anndata_logcounts.h5"
```

Comment:
Better to use temporary files in tests. I don't know how extensively we hard code file names that we are creating elsewhere, but it is probably something we should look to clean up as we see it.
```suggestion
  new_anndata_file <- tempfile(fileext = ".h5")
```